examples/src/main/resources/ssl/server.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIEpQIBAAKCAQEA2FADBs38uSLTypNJhuHKRfKnxQ3Coj+St0boSRSdfiOc7pcg
```

Comment:
commit private key into repo is not a recommended, do we have other ways?

Reply:
it's an example to show how to use the SSL function, just a temporarily key